docusaurus/docs/guides/kubernetes/self-hosting/_category_.json

```diff
@@ -0,0 +1,7 @@
+{

```

Comment:
is there any other kind of hosting? when i looked at the sidebar this kinda didn't really make sense to me to see it under another menu item.

Reply:
I meant to organize the Ziti stack deployments separately from workload-related categories like tunneling non-Ziti workloads, Ziti ingress, or deploying Ziti forks as workloads.